### PR TITLE
Jeti Sensors - RF2.2.X

### PR DESCRIPTION
**Pull request by robthomson**
Added in additional sensors to work with jeti.

Currently added:

- headspeed
- tailspeed
- arming flags
- pid profile
- rate profile
- governor mode
- throttle control
- adjfunction
- adjvalue
- esc temp

From tests it seems that this is now the max number of sensors we can add.  Next steps would be a total refactor of telem to make sensors selectable.  I dont have time to do that - so this will have to suffice.

### CODE REVIEW

File: src/main/telemetry/jetiexbus.c [hunk 1]
```diff
@@ -116,7 +123,7 @@ typedef struct exBusSensor_s {
 // list of telemetry messages
 // after every 15 sensors a new header has to be inserted (e.g. "BF D2")
 const exBusSensor_t jetiExSensors[] = {
-    {"BF D1",           "",         EX_TYPE_DES,   0              },     // device descripton
+    {"Rotorflight D1",           "",         EX_TYPE_DES,   0              },     // device descripton
```
Comment (rotorflight) [inline src/main/telemetry/jetiexbus.c:126]: Please maintain the same indentation.
One intend = 4 spaces. NO TABS!

File: src/main/telemetry/jetiexbus.c [hunk 2]
```diff
@@ -132,12 +139,22 @@ const exBusSensor_t jetiExSensors[] = {
     {"GPS Speed",       "m/s",      EX_TYPE_22b,   DECIMAL_MASK(2)},
     {"GPS H-Distance",  "m",        EX_TYPE_22b,   DECIMAL_MASK(0)},
     {"GPS H-Direction", "\xB0",     EX_TYPE_22b,   DECIMAL_MASK(1)},
-    {"BF D2",           "",         EX_TYPE_DES,   0              },     // device descripton
+    {"Rotorflight D2",           "",         EX_TYPE_DES,   0              },     // device descripton
```
Comment (rotorflight) [inline src/main/telemetry/jetiexbus.c:142]: Indentation

**Review by rotorflight — CHANGES_REQUESTED** [review]
Fix indentation.

File: src/main/telemetry/jetiexbus.c (hunk 1 above)
Comment (robthomson) [reply src/main/telemetry/jetiexbus.c:126]: I suspect this was there already in the code as a tab as only thing bg changed was the bf to rotorflight.

Will adjust

File: src/main/telemetry/jetiexbus.c (hunk 2 above)
Comment (robthomson) [reply src/main/telemetry/jetiexbus.c]: Resolved.

**Comment by robthomson** [comment]
Ready for review again

File: src/main/telemetry/jetiexbus.c [hunk 3]
```diff
@@ -391,11 +435,78 @@ int32_t getSensorValue(uint8_t sensor)
...
+    case EX_PID_PROFILE:
+        return telemetrySensorValue(TELEM_PID_PROFILE);
+    break;
+
+    case EX_RATES_PROFILE:
+        return telemetrySensorValue(TELEM_RATES_PROFILE);
+    break;
+
+    case EX_GOVERNOR_MODE:
+                if (!ARMING_FLAG(ARMED)) {
```
Comment (rotorflight) [inline src/main/telemetry/jetiexbus.c:459]: Wrong indentation...

File: src/main/telemetry/jetiexbus.c [hunk 4]
```diff
@@ -391,11 +435,78 @@ int32_t getSensorValue(uint8_t sensor)
...
+    break;
+
+    case EX_ADJVALUE:
+        return getAdjustmentsRangeValue();
+    break;
+
+
+#ifdef USE_ESC_SENSOR_TELEMETRY
+    case EX_ESCTEMP:
+            escData = getEscSensorData(0);
```
Comment (rotorflight) [inline src/main/telemetry/jetiexbus.c:495]: Wrong indentation.

**Review by rotorflight — CHANGES_REQUESTED** [review]
Indentation.

**Review by rotorflight — APPROVED** [review]

**Review by rotorflight — APPROVED** [review]

**Pull request merged**